md/GLSL_Instructions_Linux.md

```diff
@@ -4,27 +4,39 @@
 
   1. Install `mpv` from repositories of your distribution, some of the common ones are mentioned here
       ### Fedora Silverblue
-        1. Install the RPM-Fusion "free" repository, if not already installed, paste in the command below
+        1. Install the RPM-Fusion repository, if not installed already, paste the below command in the terminal
 
             `sudo rpm-ostree install https://mirrors.rpmfusion.org/free/fedora/rpmfusion-free-release-$(rpm -E %fedora).noarch.rpm https://mirrors.rpmfusion.org/nonfree/fedora/rpmfusion-nonfree-release-$(rpm -E %fedora).noarch.rpm`
 
         2. Reboot and install `mpv`
         
         3. Reboot and continue to step 2
        
-        ### Fedora
+        ### Fedora Workstation/Spins
+        1. Install the RPM-Fusion repository, if not installed already, paste the below command in the terminal  
+       
+           `sudo dnf install https://mirrors.rpmfusion.org/free/fedora/rpmfusion-free-release-$(rpm -E %fedora).noarch.rpm https://mirrors.rpmfusion.org/nonfree/fedora/rpmfusion-nonfree-release-$(rpm -E %fedora).noarch.rpm`
+         
          sudo dnf install mpv
-
-        ### Ubuntu and Derivatives
+         
+   - Note: RPM Fusion command for both Fedora Workstation and Fedora Silverblue will add both "free" and "non-free" repository.  
+         To know more about these you can check this [link](https://rpmfusion.org/Configuration)
+         
+        ### Debian/Ubuntu and Derivatives
          sudo apt install mpv
+         
+   - Note: Debian and Ubuntu usually ships with the older version of mpv player in their official repositories. It is advised to check flatpak guide below as it stays upto date with the upstream. 
 
         ### Arch and Derivatives
```

Comment:
Do we really need to teach a linux user to use the Package Manager to install a software?